shoop/core/models/_service_behavior.py

```diff
@@ -168,3 +171,42 @@ def get_unavailability_reasons(self, service, source):
         groups_to_match = set(self.groups.all().values_list("pk", flat=True))
         if not bool(customer_groups & groups_to_match):
             yield ValidationError(_("Service is not available for any of the customers groups."))
+
+
+class RoundingBehavior(Enum):
+    ROUND_UP = 1
+    ROUND_DOWN = 2
+
+    class Labels:
+        ROUND_UP = _("round up")
+        ROUND_DOWN = _("round down")
+
+
+class RoundingBehaviorComponent(ServiceBehaviorComponent):
+    name = _("Rounding")
+    help_text = _("Round total order price to the nearest five cents.")
+
+    rounding_behavior = EnumIntegerField(
+        RoundingBehavior,
+        default=RoundingBehavior.ROUND_DOWN,
+        verbose_name=_("rounding behavior"))
+
+    def round_down(self, amount):
+        return (amount * 2).quantize(decimal.Decimal('.1'), rounding=decimal.ROUND_DOWN) / 2
+
+    def round_up(self, amount):
+        return (amount * 2).quantize(decimal.Decimal('.1'), rounding=decimal.ROUND_UP) / 2
```

Comment:
The rounding "quant", which seems to be 0.05 here, should be configurable: Add it as field to the model. Also the so called "nickel rounding" algorithm could be useful as a separate util. You should probably define the nickel rounding algorithm in shoop.utils.numbers. It could be something like this:
```python
def nickel_round(value, quant=Decimal('0.05'), rounding=decimal.ROUND_HALF_UP):
    """
    Round decimal value to nearest quant.

    For example:

    >>> assert nickel_round(Decimal('10.31')) == Decimal('10.30')
    >>> assert nickel_round(Decimal('10.33')) == Decimal('10.35')
    >>> assert nickel_round(Decimal('10.325')) == Decimal('10.35')
    >>> assert nickel_round(Decimal('10.3249')) == Decimal('10.30')
    >>> assert nickel_round(Decimal('10.31'), Decimal('0.125')) == Decimal('10.250')
    >>> assert nickel_round(Decimal('10.32'), Decimal('0.125')) == Decimal('10.375')
    """
    assert isinstance(value, Decimal)
    return (value / quant).quantize(1, rounding=rounding) * quant
```